codefresh.yml

```diff
@@ -0,0 +1,74 @@
+# More examples of Codefresh YAML can be found at
+# https://codefresh.io/docs/docs/yaml-examples/examples/
+
+version: "1.0"
+# Stages can help you organize your steps in stages
+stages:
+  - "clone"
+  - "test"
+  - "release"
+
+steps:
+  clone:
+    title: "Cloning repository"
+    type: "git-clone"
+    repo: "a2i2/surround"
+    # CF_BRANCH value is auto set when pipeline is triggered
+    # Learn more at codefresh.io/docs/docs/codefresh-yaml/variables/
+    revision: "${{CF_BRANCH}}"
+    git: "github"
+    stage: "clone"
+
+  test:
+    title: "Run Python tests"
+    type: "freestyle" # Run any command
+    image: "python:${{PYTHON_VERSION}}" # The image in which command will be executed
```

Comment:
I'd just hard-code the Python version here rather than loading it from a project variable.

The reason for this is that if we want to upgrade to a different Python version, it's likely that there will be several other places in the repo that need to be updated (e.g. `setup.py`, `README.md`, `docker-compose.yml`), so we could update this at the same time with a simple find-and-replace as opposed to an additional manual step.